Header header
std_msgs/UInt8 njoys
sensor_msgs/Joy[njoys] joys